config/jobs/kubernetes/sig-node/dynamic-resource-allocation.jinja

```diff
@@ -0,0 +1,115 @@
+{%- if beginning %}
```

Comment:
Can this file be moved into a `templates` directory, as in kops?

When I look at the PR sidebar, I currently see four files with the identical `dynamic-resource-allocation...` as name. Even if we shorten that to `dra-`, keeping the source file separate would make it stand out more.

Reply:
Sure, it can be moved. Should I move .conf file as well?

Personally, I'd prefer flat structure with shorter names, e.g.
dra.conf
dra.jinja
dra-canary.yaml
dra-pull.yaml
dra-ci.yaml

And I hope that this approach can be used for all sig-node jobs and the final list of files will be something like this:
jobs.conf
jobs.jinja
jobs-canary.yaml
jobs-pull.yaml
jobs-ci.yaml